### PR TITLE
Better error message on duplicated samples in the templates

### LINKED ISSUE #1341: Ugly error on prep template duplicate samples
```
An error occurred parsing the prep template: prep_template_CP_SCRIPT.txt
Error running SQL query in queue CREATE_PREP_TEMPLATE_375_142905488: INSERT INTO qiita.prep_template_sample (prep_template_id, sample_id) VALUES (%s, %s)
ARGS: [723L, '375.C1A']
Error: duplicate key value violates unique constraint "idx_common_prep_info"
DETAIL: Key (prep_template_id, sample_id)=(723, 375.C1A) already exists.
```
This should be caught and shown as "Duplicate sample names found!"

### PR DESCRIPTION
**Pull request by josenavas**
Fixes #1341

Example of output:
screen shot 2015-08-07 at 12 06 54 pm

### CODE REVIEW

File: qiita_db/exceptions.py [hunk 1]
```diff
@@ -72,6 +72,14 @@ def __init__(self, repeated_headers):
                      "header(s): %s." % ', '.join(repeated_headers),)
 
 
+class QiitaDBDuplicateSamplesError(QiitaDBError):
+    """Exception for error when a MetadataTemplate has duplicate columns"""
+    def __init__(self, repeated_samples):
+        super(QiitaDBDuplicateSamplesError, self).__init__()
+        self.args = ("Duplicate samples found in MetadataTemplate. Repeated "
```
Comment (squirrelo) [inline qiita_db/exceptions.py]: Can remove the "Repeated samples:" from this to save room, since it's saying the same thing as the first sentence.

Comment (josenavas) [inline qiita_db/exceptions.py]: Done

**Comment by antgonza** [comment]
:+1:

**Comment by josenavas** [comment]
@squirrelo can you merge if there are no more comments?

**Pull request merged**